components/profile/network-item/body.view.tsx

```diff
@@ -0,0 +1,136 @@
+import { useTranslation } from "next-i18next";
+
+import ArrowDown from "assets/icons/arrow-down";
+import ArrowUp from "assets/icons/arrow-up";
+import ArrowUpRight from "assets/icons/arrow-up-right";
+
+import Button from "components/button";
+import ResponsiveWrapper from "components/responsive-wrapper";
+
+import { FlexColumn, FlexRow } from "../wallet-balance";
+import NetworkItemAmountView from "./amount.view";
+
+interface NetworkItemBodyViewProps {
+  isCollapsed: boolean;
+  handleNetworkLink: () => void;
+  type: "network" | "voting";
+  amount: string | number;
+  symbol: string;
+  isNetworkVariant: boolean;
+  primaryColor?: string;
+  isNetworkType: boolean;
+  handleToggleCollapse: () => void;
+}
+
+export default function NetworkItemBodyView({
+  isCollapsed,
+  type,
+  amount,
+  symbol,
+  isNetworkVariant,
+  primaryColor,
+  handleNetworkLink,
+  isNetworkType,
+  handleToggleCollapse,
+}: NetworkItemBodyViewProps) {
+  const { t } = useTranslation(["profile"]);
+
+  function ArrowComponent() {
+    if (isCollapsed) return <ArrowDown width={10} height={8} />;
+
+    return <ArrowUp width={10} height={8} />;
+  }
+
+  function NetworkLinkIconButton({ className = "" }) {
+    return (
+      <div
+        className={`${className} py-0 mt-1 ms-4 cursor-pointer border border-gray-700 bg-gray-850 border-radius-4`}
+        onClick={handleNetworkLink}
+      >
+        <ArrowUpRight />
+      </div>
+    );
+  }
+
+  function RenderAmount() {
+    return (
+      <NetworkItemAmountView
+        amount={amount}
+        symbol={symbol}
+        isNetworkVariant={isNetworkVariant}
+        type={type}
+        primaryColor={primaryColor}
+      />
+    );
+  }
+
+  if (isNetworkType)
```

Comment:
this should be the controller who decides, not the view -- we need two components for each and the controller decides what to show